app/views/layouts/admin_layout.html.erb

```diff
@@ -55,6 +55,12 @@
         } %>
       <% end %>
 
+      <% if flash["danger"].present? %>
```

Comment:
Rather than adding this here, the controller could be updated to set `flash[:alert]` instead. That would follow the changes made to use the design system: https://github.com/alphagov/collections-publisher/pull/674/commits/7a072d84ad4f7cda567a06aa425f8de7583fa85d#diff-5f9771f9b5dbe7b3a9da755f7b5dc9bfL100

Reply:
That was my first approach, then I realised there are lots of instances of `flash[:danger]` elsewhere in the repository, so thought that approach might require changes in lots of places.

But come to think of it, other controllers don't set `layout 'admin_layout'` so it probably isn't an issue that, say, ListItemsController uses `flash[:danger]`. So I'll go back and do as you've suggested 👍

Reply:
Yeah, the other pages that use `flash[:danger]` are still using the old admin layout.